testsuite/basic10-fork-no-constraint/run-tests.tpl

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+{% include 'common-run.sh.tpl' %}
+
+s2e run -n {{ project_name }}
+
+# Make sure this test case generates exactly 4 states.
+# state 0 already exists, state 1~3 are spawned afterwards.
+grep -q "to state 1" $S2E_LAST/debug.txt
+grep -q "to state 2" $S2E_LAST/debug.txt
+grep -q "to state 3" $S2E_LAST/debug.txt
+! grep -q "to state 4" $S2E_LAST/debug.txt
+
+# State 1 should be spawned by s2e_fork.
+grep -zoP  "    state 0\n    state 1" $S2E_LAST/debug.txt
+
+# State 2 should be spawned by the else branch of State 0.
+grep -zoP  "    state 2\n    state 0" $S2E_LAST/debug.txt
+
+# State 3 should be spawned by the else branch of State 1.
+grep -zoP  "    state 3\n    state 1" $S2E_LAST/debug.txt
+
```

Comment:
From s2e-last/debug.txt.

The following lines are printed after `s2e_fork()` is called.
```
6 [State 0] Forking state 0 at pc = 0x80061359 at pagedir = 0xe428000
    state 0
    state 1
```

The following lines are printed after state 0 is forked, producing state 2.
```
6 [State 0] Forking state 0 at pc = 0x800613ab at pagedir = 0xe428000
    state 2
    state 0
```

The following lines are printed after state 1 is forked, producing state 3.
```
7 [State 1] Forking state 1 at pc = 0x800613ab at pagedir = 0xe428000
    state 3
    state 1
```

Reply:
Don't grep those. It's enough to grep your own messages.